docs/java/overview.asciidoc

```diff
@@ -0,0 +1,42 @@
+== Overview
+
+Official low-level client for Elasticsearch. Allows to communicate with an
+Elasticsearch cluster through http. Compatible with any elasticsearch version.
+
+=== Features
+
+The low-level client's features include:
+
+* minimal dependencies (apache http client and its transitive dependencies)
```

Comment:
Maybe 
Apache https://hc.apache.org/httpcomponents-asyncclient-dev/httpasyncclient/project-summary.html[httpasyncclient], including https://commons.apache.org/proper/commons-codec/[commons-codec] and https://commons.apache.org/proper/commons-logging/[commons-logging])